VideoLocker/default_config/config.yaml

```diff
@@ -3,3 +3,4 @@ ENVIRONMENT_DISPLAY_NAME: 'Localhost'
 PLATFORM_NAME: 'Your platform name here'
 PLATFORM_DESTINATION_NAME: 'example.com'
 FEEDBACK_EMAIL_ADDRESS: 'support@example.com'
+ORGANIZATION_CODE: ''
```

Comment:
We usually define our config flags here. https://openedx.atlassian.net/wiki/display/MA/App+Configuration+Flags

But now that I think about it, maybe we should revisit the default config. I'll follow up on this.